tailwind.config.js

```diff
@@ -152,11 +155,13 @@ module.exports = {
         '428px': '26.75rem',
         '432px': '27rem',
         '456px': '28.5rem',
+        '476px': '29.75rem',
         '478px': '29.875rem',
         '492px': '30.75rem',
         '526px': '32.875rem',
         '540px': '33.75rem',
         '553px': '34.563rem',
+        '556px': '34.75rem',
```

Comment:
No need to add new sizes here since you used classes with square braces 'w-[556px]' so it it using a jit mode. I see other sizes are used without braces 'w-292px'

Reply:
I have updated. Please check again